packages/react-components/react-search/docs/MIGRATION.md

```diff
@@ -0,0 +1,94 @@
+| input className              | `input.className?: `                                        |                                         |
+| aria label                   | `'aria-label'?: string`                                     | `'aria-label'?: string`                 |
+| clearable                    | `clearable?: boolean`                                       | `dismiss` slot                          |
+| style overrides              | `styles?: ComponentSlotStyles<...>`                         | className, slot classNames              |
+| accessibility                | individual props                                            | individual props                        |
+
+```ts
+// packages/fluentui/react-northstar/src/utils/htmlPropsUtils.tsx
+type SupportedIntrinsicInputProps = {
+  [K in HtmlInputProps]?: K extends keyof JSX.IntrinsicElements['input'] ? JSX.IntrinsicElements['input'][K] : any;
+};
+// HtmlInputProps: a subset of React and HTML native props
+
+// packages/fluentui/react-northstar/src/components/Box/Box.tsx
+interface BoxProps extends UIComponentProps<BoxProps>, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
+}
+```
+
+## Migrations from v0
+
+| Prop/concept                 | v8                                                    | v9                                      |
+| ---------------------------- | ----------------------------------------------------- | --------------------------------------- |
+| imperative API               | `componentRef`                                        | n/a                                     |
+| supported native props       | `React.AllHTMLAttributes` (input or textarea)         | `React.InputHTMLAttributes`             |
+| setting native props on root |                                                       | `root` slot                             |
+| root element access          | `elementRef`                                          | `ref` on `root` slot                    |
+| primary element access       | not possible                                          | top-level `ref`                         |
+| multiline mode               | `multiline?: boolean`                                 | separate component                      |
+| underlined                   | `underlined?: boolean`                                | `appearance`                            |
+| borderless                   | `borderless?: boolean`                                | not supported?                          |
+| fluid (full width)           | default behavior                                      | default behavior                        |
+| inline                       | n/a (use styling)                                     | `inline?: boolean`                      |
```

Comment:
Same here, we no longer have `inline` as a prop, so this would be "custom styles"